src/main/webapp/app/overview/course-lectures/course-lecture-details.component.html

```diff
@@ -19,6 +19,11 @@ <h4>
                     </span>
                 </h4>
             </div>
+            <div class="col-auto wrapper-btn-manage-lecture" *ngIf="!!this.lecture?.course?.isAtLeastTutor">
```

Comment:
```suggestion
            <div class="col-auto d-flex align-items-center" *ngIf="!!this.lecture?.course?.isAtLeastTutor">
```
I think using bootstrap classes is more clear than defining custom css class

Reply:
Thanks, that't much smarter approach 👍🏻 Done